lib/util/cache/repository/common.ts

```diff
@@ -1,50 +1,16 @@
 import is from '@sindresorhus/is';
-import type {
-  RepoCacheRecordV10,
-  RepoCacheRecordV11,
-  RepoCacheRecordV12,
-} from './types';
+import type { RepoCacheRecord } from './types';
 
 // Increment this whenever there could be incompatibilities between old and new cache structure
-export const CACHE_REVISION = 12;
+export const CACHE_REVISION = 1;
```

Comment:
Why not just keep incrementing? I know you can differentiate, but I'm not sure I like it